utils/content.ts

```diff
@@ -1,21 +1,41 @@
 import fs from 'node:fs';
 import path from 'node:path';
```

Comment:
updated the imports from `import fs form 'fs'` to `import fs form 'node:fs'`. The same for `path`

Reply:
All of these node: changes you added can be added to another new separate  PR to the upstream vickonrails repo - then our repo doesn't diverge as much